wp-content/plugins/core/src/Theme/Image_Sizes.php

```diff
@@ -7,6 +7,8 @@
 class Image_Sizes {
 	const CORE_FULL      = 'core-full';
 	const CORE_MOBILE    = 'core-mobile';
+	const CORE_SQUARE    = 'core-square';
+	const CORE_LANDSCAPE = 'core-landscape';
```

Comment:
Naming is always fun! We've been starting to follow a pattern around naming more based around aspect ratio these days (with the exception of core-full & core-mobile), so instead of: `CORE_SQUARE = 'core-square';` it would be `	const IMAGE_16X9 = 'image-16x9';`

This is a really handy tool I use all the time: https://andrew.hedges.name/experiments/aspect_ratio/

Reply:
Do you mean replace both for IMAGE_1X1 and IMAGE_16X9?

Reply:
Yep